### PR TITLE
UI styling

### PR DESCRIPTION
**Pull request by lpissinali**
- Adding search fields to unavailable and registered screens #16 
- Styling modals
- Converting input fields to static text on registering new ens name #7 
- Adding terms page (missing styling) #2

### CODE REVIEW

**Comment by jeluard** [comment]
@lpissinali Could you link any relevant issues being fixed / addressed ?

File: app/components/ens/registerSubDomain.js [hunk 1]
```diff
@@ -102,28 +115,43 @@ const InnerForm = ({
...
-            style={{ marginTop: '10px' }}
-            placeholder="Ethereum Address"
-            value={values.address}
-            onChange={handleChange}
-            onClick={() => setFieldValue('address', '')}
-            required
-            wide />
-        </Field>
+
+        <DisplayBox displayType='Your wallet address' pubKey={values.address} />
```
Comment (bgits) [inline app/components/ens/registerSubDomain.js]: Are we only going to display the address and not allow the user to change it?

Comment (lpissinali) [reply app/components/ens/registerSubDomain.js]: According to Rachel, there's no need of input fields here.

**Comment by bgits** [comment]
added a fix for: https://github.com/status-im/ens-usernames/issues/31

**Pull request merged**